changelog/7492.improvement.md

```diff
@@ -0,0 +1,3 @@
+Added `./docker/Dockerfile_pretrained_embeddings_spacy_en_ja` to include Spacy's Japanese pre-trained model `ja_core_news_md`.
+
+Updated `spacy` to `2.3.2` and `poetry.lock` during the process of adding Japanese pre-trained model.
```

Comment:
This won't be necessary after you've updated your branch with the latest changes from `main` as spacy is already upgraded to version `3.2`.